src/javascript/Switch.js

```diff
@@ -52,6 +56,9 @@ export class Switch {
 
         this.element.classList.add("_simple-switch-checkbox");
         this.track.classList.add("_simple-switch-track");
+        if (this.disabled) {
+            this.track.classList.add("disabled")
```

Comment:
NIT: "disabled" is pretty simple and likely to be fairly common. It might be better to use something like `_simple-switch-disabled` to prevent collisions

(this is presuming that you don't remove this in favor of the getter / setter pattern I suggested)